mitosheet/mitosheet/code_chunks/export_to_file_code_chunk.py

```diff
@@ -12,11 +12,24 @@
 
 from mitosheet.transpiler.transpile_utils import param_dict_to_code
 
+from mitosheet.excel_utils import get_column_from_column_index
+
 # This is a helper function that generates the code for formatting the excel sheet
 def get_format_code(state: State) -> list:
     code = []
     formats = state.df_formats
-    for sheet_name, format in zip(state.df_names, formats):
+    for sheet_index, (sheet_name, format) in enumerate(zip(state.df_names, formats)):
+        # We need to convert the column IDs to column letters for conditional formats to export to excel
+        conditional_formats = format.get('conditional_formats')
+        column_ids = state.column_ids.get_column_ids(sheet_index=sheet_index)
+        if conditional_formats is not None:
+            for conditional_format in conditional_formats:
+                # Create new object to store the columns in the excel format
+                conditional_format['columns'] = []
+                for column_id in conditional_format.get('columnIDs'):
+                    column = get_column_from_column_index(column_ids.index(column_id))
```

Comment:
You're missing a step here! Remember that we need to go from ColumnID => ColumnHeader => int => excel column. 

Notably, ColumnID is a Mito internal concept that allows us to have a static id for a specific column, even if you rename or sort it. It's useful in a couple of places, but generally it's just really nice to be able to refer to a column by a specific ID over the full lifecycle of an analysis. 

This means that the column_ids in the the column_ids list that you have -- there are no explicit ordering constraints on it. This is just whatever order the column ids happened to be in when they were first created. 

Instead, here's what you want to do:
```
for column_id in conditional_format.get('columnIDs'):
     column_header = state.column_ids.get_column_header_by_id(sheet_index, column_id)
     column_index = state.dfs[sheet_index].columns.tolist().index(column_header)
     .... 
```
Let me know if this doesn't make sense. The primary thing to callout is that the _only place_ you can reliably get the index of a column in a dataframe is by checking the dataframe, which requires the actual column header, which you can get from the column id. 

Let me know if any of this is unclear! Maybe you could write up your understanding of this as a longer comment in the `ColumnIDMap` class - that way future Mito programmers can figure it out easier - and we can debug your understanding more if necessary :)

Reply:
Oh ok good to know! I'll try to add some comments to clarify that in the `ColumnIDMap` - From looking at the comments in that file, I suspected that I wasn't getting the index correctly. I do think it was kind of hard to wrap my head around where exactly to get the actual index of the column, so it'd be good to document that a bit

Reply:
Added a short comment in that class, but lmk if you think I should elaborate / add more in other places